requirements.txt

```diff
@@ -4,5 +4,5 @@ Fabric==0.9.0
 nose==1.0.0
 rdflib==3.2.1
 argparse==1.1
--e git://github.com/mattbasta/fastchardet#egg=fastchardet
+fastchardet
```

Comment:
Thanks for the quick reply! I'll do that. However, do you use to amend the changes and/or squash?